src/core/worker.js

```diff
@@ -436,6 +437,44 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
       });
     });
 
+    handler.on('GetRawObject', function wphGetRawObject(data) {
+      function clone(obj) {
+        if (!obj || isBool(obj) || isNum(obj) || isString(obj)) {
+          return obj;
+        } else if (isName(obj)) {
+          return { name: obj.name };
+        } else if (isDict(obj) || isArray(obj)) {
+          return obj.map(clone);
```

Comment:
This doesn't look entirely correct, compare with the original PR:
https://github.com/mozilla/pdf.js/pull/5117/files#diff-780c33139a29f60c0cc45b80fb05f900R409 and https://github.com/mozilla/pdf.js/pull/5117/files#diff-780c33139a29f60c0cc45b80fb05f900R416.